crates/lakekeeper/src/catalog/tabular.rs

```diff
@@ -36,29 +36,51 @@ macro_rules! list_entities {
                     query,
                 )
                 .await?;
+                let namespace_id =
+                    C::namespace_to_id($warehouse_id, &namespace, trx.transaction()).await;
+                let mut can_list_everything = false;
+                if let Ok(Some(n)) = namespace_id {
+                    can_list_everything = authorizer
+                        .is_allowed_namespace_action(
+                            &request_metadata,
+                            NamespaceId::from(*n),
+                            CatalogNamespaceAction::CanListEverythingInNamespace,
+                        )
+                        .await
+                        .unwrap_or(false);
```

Comment:
Same as above we can't just unwrap any error to false. Maybe OpenFGA is down and this is actually a 500 error that we must popagate. So lets use `?` instead.